docs/core/kernel-code.md

```diff
@@ -0,0 +1,68 @@
+
+# List of all kernel functions
+
+| Module                    | Function      | Notes         | Used by       |
+| ------------------------- | ------------- | ------------- | ------------- |
+| `Elm.Kernel.Basics`       | `add`         | [1](###1)     | `Basics.add`  |
+|                           | `sub`         | [1](###1)     | `Basics.sub`  |
+|                           | `mul`         | [1](###1)     | `Basics.mul`  |
+|                           | `fdiv`        | [1](###1)     | `Basics.fdiv` |
+|                           | `idiv`        | [1](###1)     | `Basics.idiv` |
+|                           | `pow`         | [1](###1)     | `Basics.pow`  |
+|                           | `pow`         | todo...       |
+|                           | `remainderBy` | todo...       |
+|                           | `modBy`       | todo...       |
+|                           | `pi`          | todo...       |
+|                           | `e`           | todo...       |
+|                           | `cos`         | todo...       |
+|                           | `sin`         | todo...       |
+|                           | `tan`         | todo...       |
+|                           | `acos`        | todo...       |
+|                           | `asin`        | todo...       |
+|                           | `atan`        | todo...       |
+|                           | `atan2`       | todo...       |
+|                           | `toFloat`     | todo...       |
+|                           | `truncate`    | todo...       |
+|                           | `isInfinite`  | todo...       |
+|                           | `ceiling`     | todo...       |
+|                           | `floor`       | todo...       |
+|                           | `round`       | todo...       |
+|                           | `sqrt`        | todo...       |
+|                           | `log`         | todo...       |
+|                           | `isNaN`       | todo...       |
+|                           | `not`         | todo...       |
+|                           | `and`         | todo...       |
+|                           | `or`          | todo...       |
+|                           | `xor`         | todo...       |
+| `Elm.Kernel.Bitwise`      | `and`             | todo...   |
+|                           | `or`              | todo...   |
+|                           | `xor`             | todo...   |
+|                           | `complement`      | todo...   |
+|                           | `shiftLeftBy`     | todo...   |
+|                           | `shiftRightBy`    | todo...   |
+|                           | `shiftRightZfBy`  | todo...   |
+| `Elm.Kernel.Char`         | todo...       |
+| `Elm.Kernel.Debug`        | todo...       |
+| `Elm.Kernel.JsArray`      | todo...       |
+| `Elm.Kernel.List`         | todo...       |
+| `Elm.Kernel.Platform`     | todo...       |
```

Comment:
```suggestion
| `Elm.Kernel.Platform`     | `worker`              |                       | `Platform.worker`                                                                                                 |
|                           | `initialize`          |                       | `Elm.Kernel.Browser.document`, `Elm.Kernel.Browser.element` in `elm/browser`                                      |
|                           | `registerPreload`     | [Preload](###Preload) | Never used.                                                                                                       |
|                           | `setupEffects`        |                       | `Elm.Kernel.Platform.Initialize`                                                                                  |
|                           | `createManager`       |                       | Compiler generated JavaScript for event managers.                                                                 |                                                                                                                                                   |
|                           | `instantiateManager`  |                       | `Elm.Kernel.Platform.setupEffects`                                                                                |
|                           | `sendToApp`           |                       | `Platform.sendToApp`                                                                                              |
|                           | `sendToSelf`          |                       | `Platform.sendToSelf`                                                                                             |
|                           | `leaf`                |                       | `Elm.Kernel.Platform.outgoingPort`, `Elm.Kernel.incomingPort`, Compiler generated JavaScript for event managers.  |
|                           | `batch`               |                       | `Platform.Cmd.batch`, `Platform.Sub.batch`                                                                        |
|                           | `map`                 |                       | `Platform.Cmd.map`, `Platform.Sub.map`                                                                            |
|                           | `dispatchEffects`     |                       | todo...                                                                                                           |
|                           | `gatherEffects`       |                       | todo...                                                                                                           |
|                           | `toEffect`            |                       | todo...                                                                                                           |
|                           | `insert`              |                       | todo...                                                                                                           |
|                           | `checkPortName`       |                       | todo...                                                                                                           |
|                           | `outgoingPort`        |                       | todo...                                                                                                           |
|                           | `outgoingPortMap`     |                       | todo...                                                                                                           |
|                           | `setupOutgoingPort`   |                       | todo...                                                                                                           |
|                           | `incomingPort`        |                       | todo...                                                                                                           |
|                           | `incomingPortMap`     |                       | todo...                                                                                                           |
|                           | `setupIncomingPort`   |                       | todo...                                                                                                           |
|                           | `export`              |                       | todo...                                                                                                           |
|                           | `mergeExportsProd`    |                       | todo...                                                                                                           |
|                           | `mergeExportsDebug`   |                       | todo...     
```

Reply:
Work in progress for the `Platform` kernel module - possibly the most interesting one.